scripts/install/opt/seagate/cortx/motr/libexec/m0_ha_sim

```diff
@@ -195,6 +279,7 @@ usage()
         echo "[OPTIONS]"
         echo "-t, --type        : Type of operation (drive/repair/rebalance/repreb)"
         echo "-o, --ops         : Sub operation of type (abort/pause/resume)"
+        echo "-r, --recover-type: Type of recover / rebalance (SNS (Default) / DIX)"
```

Comment:
@sanjognaik  Pls remove "rebalance" word from "-r" usage msg. And what will happen if I provide  "-t drive -r sns"?

Reply:
Removed word 'rebalance' from the usage message for '-r' option. 

> And what will happen if I provide "-t drive -r sns"?

In this case only -t drive option will be considered and -r will be ignored. 
- The option -t expects device id or device path or device status. It ignores all other options.
- The option -r is considered only when type (-t) is repair / rebalance / repreb. Otherwise it will be ignored.